### PR TITLE
stone-soup: depends on pyyaml

### PR DESCRIPTION
**Pull request by singingwolfboy**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/stone-soup.rb [hunk 1]
```diff
@@ -23,26 +21,19 @@ class StoneSoup < Formula
...
-  resource "PyYAML" do
-    url "https://files.pythonhosted.org/packages/36/2b/61d51a2c4f25ef062ae3f74576b01638bebad5e045f747ff12643df63844/PyYAML-6.0.tar.gz"
-    sha256 "68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2"
-  end
-
   def install
     ENV.cxx11
     ENV.prepend_path "PATH", Formula["python@3.10"].opt_libexec/"bin"
     python3 = "python3.10"
     ENV.prepend_create_path "PYTHONPATH", buildpath/"vendor"/Language::Python.site_packages(python3)
```
Comment (danielnachun) [inline Formula/stone-soup.rb:35]: Should we also get rid of all this?  Does this even need to depend on `python@3.10` at all anymore?

Comment (chenrui333) [reply Formula/stone-soup.rb:35]: agree, maybe line 39 to line 41 can be removed

Comment (chenrui333) [reply Formula/stone-soup.rb:35]: I can give a try in my local

Comment (chenrui333) [reply Formula/stone-soup.rb]: good to be removed

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**